src/data/recycling-facts.json

```diff
@@ -0,0 +1,76 @@
+[
+  "Recycling is a $200 billion industry in the U.S.",
+  "Recycling generates 7-10 more jobs that landfills and waste to energy plants.",
+  "Recycling properly creates valuable resources for U.S. manufacturing and can become a highly valuable export to countries such as China and India.",
+  "Recycling conserves finite natural resources - this is critical as ​population continues to grow exponentially.",
+  "Recycling conserves fresh water up to 95% in the mining and manufacturing process for many materials.",
```

Comment:
I think what this is trying to say is "Recycling conserves fresh water, up to 95%, during the mining and manufacturing process..."